pkgs/development/libraries/cjson/default.nix

```diff
@@ -0,0 +1,23 @@
+{stdenv, fetchFromGitHub, cmake}:
```

Comment:
```suggestion
{ lib, fetchFromGitHub, cmake }:
```

Reply:
How does this work if the derivation uses `stdenv.mkDerivation`?